R/quosure-constructor.R

```diff
@@ -45,8 +33,10 @@ setMethod(
   "new_quosure",
   signature = c(env = "environment", code = "expression"),
```

Comment:
please put this method first as this is the "main" constructor that all the others are using